apps/service_define/lib/define/view_state.ex

```diff
@@ -20,9 +23,8 @@ defmodule Define.ViewState do
   end
 
   @impl true
-  def handle_call({:event, type, payload}, _from, state) do
-    new_state = update_state(state, type, payload)
-    {:reply, new_state, new_state}
+  def handle_call({:event, _type, _payload}, _from, state) do
+    {:reply, state, state}
```

Comment:
I assume we continue to keep this module for what we know to be coming? We might want to document that for now.